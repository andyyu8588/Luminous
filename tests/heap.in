import Heap

function simpleNumberHash(x) {
	return x;	
}

numHeap = Heap(simpleNumberHash);

numHeap.add(1);
numHeap.add(17);
numHeap.add(11);
numHeap.add(0);
numHeap.add(8);

print(numHeap.size());
print(numHeap.deleteMin());
print(numHeap.deleteMin());
print(numHeap.deleteMin());
print(numHeap.deleteMin());
print(numHeap.deleteMin());