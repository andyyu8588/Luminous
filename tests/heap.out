5
0
1
8
11
17
